src/lib/serviceRoutinesV2/deleteEntity.cpp

```diff
@@ -79,19 +88,27 @@ std::string deleteEntity
 
     if ((scc != SccOk) && (scc != SccNone))
     {
-      ErrorCode error;
+      OrionError oe;
 
+      ciP->httpStatusCode = scc;
       if (scc == SccContextElementNotFound)
       {
-        error.fill("NotFound", "The requested entity has not been found. Check type and id");
+          oe.code = scc;
+          oe.reasonPhrase ="NotFound";
+          oe.details = "The requested entity has not been found. Check type and id";
+      }
+      else if (scc == SccInvalidParameter)
+      {
+          oe.code = SccContextElementNotFound;
+          oe.reasonPhrase ="NotFound";
+          oe.details = "Attribute not found";
+          ciP->httpStatusCode = SccContextElementNotFound; // We don't want a 472
       }
       else
       {
-        error.fill(sc);
+          oe.code = scc;
```

Comment:
I would use what's in 

    upcrsP->contextElementResponseVector[ix]->statusCode